src/core/packages/chrome/layout/core-chrome-layout-components/application/layout_application.styles.ts

```diff
@@ -9,21 +9,33 @@
 
 import { css } from '@emotion/react';
 import { layoutVar, layoutLevels } from '@kbn/core-chrome-layout-constants';
+import { euiOverflowScroll } from '@elastic/eui';
 import type { EmotionFn } from '../types';
 
-const root: EmotionFn = ({ euiTheme }) =>
+const root: EmotionFn = (useEuiTheme) =>
   css`
     grid-area: application;
-    height: 100%;
-    position: relative;
-    width: 100%;
+
+    height: calc(100% - ${layoutVar('application.marginBottom')});
```

Comment:
I think the original CSS might have been redundant, and explicit height and width might not be needed here at all.

I'll create an issue for myself to double check this if can be simplified